include/matx/generators/random.h

```diff
@@ -518,6 +539,9 @@ template <typename T, int RANK> class randomTensorView_t {
           if constexpr (std::is_same_v<T, float>) {
             curandGenerateUniform(gen_, &val, 1);
           }
+          else if constexpr (std::is_same_v<T, uint32_t>) {
```

Comment:
Doesn't this need a `randomi` function?

Reply:
this has an older version of the random changes. I just needed a working version for a test function. I'll update this one once I have the other PR completed.